### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by watkin36** [watkin36/codelab-ruby-laptop-setup#1]
Hey @badsauce, can you take a look at this? It's [hosted here](https://pacific-reaches-6707.herokuapp.com/) and meets the following [project](http://lansingcodelab.com/lessons/ruby-laptop-setup/1) criteria:
- [x] be in a GitHub repo (since you're getting the app from Heroku, you'll have to go to the project folder, then remove their origin with `git remote rm origin`, and finally once again add the correct GitHub repo with `git remote add origin https://github.com/YOUR-USERNAME/codelab-ruby-laptop-setup.git`)
- [x] be hosted on Heroku

**Comment by Badsauce** [comment]
Hey Scott, you got the code up on Heroku successfully but you still need to push the code to this github repo. You can do this by following the instructions in the first step or by adding another remote: https://help.github.com/articles/adding-a-remote/

**Comment by watkin36** [comment]
Ok, I've tried it again using the "git remote add origin".  Did that work?

**Comment by Badsauce** [comment]
Yeah, you can see your success because there is code in the repo - https://github.com/watkin36/codelab-ruby-laptop-setup

Nice work :shipit:
